source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container-dark.css

```diff
@@ -10,7 +10,7 @@
 
 .berlin-header {
```

Comment:
@paulcam206 @licanhua Should we update the header styling to use margins as well?

Reply:
probably...